semantics.json

```diff
@@ -137,7 +137,14 @@
         "name": "displaySubmitButton",
         "type": "boolean",
         "label": "Display Submit button on summary page",
-        "description": "When enabled, will display a submit button on the summary page. Toggling this settign will not effect XAPI firing.",
+        "description": "When enabled, will display a submit button on the summary page. Toggling this setting will not effect XAPI firing.",
+        "default": false
+      },
+      {
+        "name": "displayRestartButton",
+        "type": "boolean",
+        "label": "Display restart button on summary page",
+        "description": "When enabled, will display a restart button on the summary page.",
         "default": false
```

Comment:
Yes, because sasha doesn't want it on existing interactive books, otherwise would have to go through each one manually to set false, which would lose user save data